### ISSUE TITLE
TS4023 for exported type instance

### ISSUE THREAD

**Issue by Geordi7** [badrap/valita#16]
In my TSConfig I set `declaration` to true, and then I started getting TS4023 Errors for almost every exported type.

It appears that there is some issue with the fact that `ObjectType` is not an `export`, and so typescript cannot create declarations for instances that are defined with it.

```typescript

import * as v from '@badrap/valita'

export const T = v.object({a: v.string()}); // TS4023

```

This is my tsconfig.json:

```json
{
    "compilerOptions": {
        "target": "esnext",
        "moduleResolution": "node",
        "outDir": "dist",
        "strict": true,
        "noImplicitAny": true,
        "declaration": true,
    },
    "include": ["src/**/*"]
}
```

**Comment by jviide** [comment]
Incidentally I bumped into the same problem and published v0.0.19 to fix this a couple of days ago. Does updating to v0.0.19 fix the problem for you?

**Comment by Geordi7** [comment]
Fixed in v0.0.19!

**Comment by jviide** [comment]
Great, thanks 🙂
